service/src/main/java/uk/gov/hmcts/reform/fpl/service/OrganisationService.java

```diff
@@ -65,7 +69,17 @@
         try {
             return ofNullable(organisationApi.findUserOrganisation(requestData.authorisation(),
                 authTokenGenerator.generate()));
+        } catch (FeignException.NotFound | FeignException.Forbidden ex) {
+            log.error("User not registered in MO", ex);
+            return Optional.empty();
+        }
+    }
 
+    public Optional<Organisation> findManagedOrganisation(String organisationId) {
+        try {
+            String userToken = idamClient.getAccessToken(userConfig.getUserName(), userConfig.getPassword());
+            return ofNullable(organisationApi.findManagedUserOrganisation(userToken,
+                authTokenGenerator.generate(), organisationId));
```

Comment:
the log in catch is not correct, it is not that user is not registered, method is executed as system user that for sure is not registered. It means organisation not found.
So can be like log.error("Organisation {} not registered", organisationId)